summit_blood_samples/blood_sample/urls.py

```diff
@@ -1,5 +1,10 @@
 from django.urls import path
 from .views import *
+from .upload_views import *
```

Comment:
Have views just been reorganised into separate files (and code tidied) or have any new views been added?

Reply:
@jonc125 We Just reorganized the code as the views file is huge.